src/libraries/Common/src/System/CodeDom/CodeTypeReference.cs

```diff
@@ -283,7 +285,7 @@ public CodeTypeReference(string typeName, params CodeTypeReference[] typeArgumen
 
 #if CODEDOM
         public CodeTypeReference(CodeTypeParameter typeParameter) :
-            this(typeParameter?.Name)
+            this(typeParameter?.Name!)
```

Comment:
seems `public CodeTypeReference(string? typeName)` on row 78 should allow nullable instead
```suggestion
            this(typeParameter?.Name)
```